src/adapters/aave-v2/tests/snapshots/ethereum.profits.json

```diff
@@ -22,7 +22,6 @@
               "name": "Maker",
               "symbol": "MKR",
               "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2/logo.png",
-              "balance": "10.019229339844049763",
```

Comment:
This was being leaked into the profits when using the spread operator on the result from balances assuming there was only metadata.

Reply:
nice catch